libutils/json.c

```diff
@@ -2019,7 +2019,9 @@ static JsonParseError JsonParseAsString(
             switch (**data)
             {
             case '\\':
+                // fall through
             case '"':
+                // fall through
```

Comment:
Since you are only falling through to `break` here, you could consider just adding `break;` instead.

Reply:
indeed i was on autopilot when i wrote this and its silly, will fix